django/tasks/register.py

```diff
@@ -0,0 +1,28 @@
+class AlreadyRegistered(Exception):
+    pass
+
+
+class TaskDoesNotExist(Exception):
+    pass
+
+
+class TaskRegistry(object):
+    def __init__(self):
+        self._registry = {}
+
+    def register(self, task):
+        if task.name in self._registry:
```

Comment:
What is the problem that they have to be unique? you can always do stuff like "app:task_name" (and probably should).